features/in_onderzoek.feature

```diff
@@ -7,134 +7,738 @@ Functionaliteit: in onderzoek
+      | fields              | ouders.<veld>                   |
+      Dan heeft de ouder met ouderAanduiding '<ouder>' de volgende 'inOnderzoek' gegevens
+      | naam                 | waarde     |
+      | <in onderzoek>       | true       |
+      | datumIngangOnderzoek | 2022-03-07 |
+      En heeft de ouder met ouderAanduiding '<andere ouder>' GEEN 'inOnderzoek' gegevens
+      En heeft de persoon GEEN 'inOnderzoek' gegevens
 
-Rule: wanneer een categorie in de bron in onderzoek is, wordt elk gegeven in het antwoord dat gevuld wordt uit een van de elementen in die categorie ook in inOnderzoek opgenomen met de waarde true
-  - de elementcode voor onderzoek eindigt op 0000
+      Voorbeelden:
+      | waarde | veld                                    | ouder  | andere ouder |
+      | 020220 | burgerservicenummer                     | ouder1 | ouder2       |
+      | 030410 | geslachtsaanduiding                     | ouder2 | ouder1       |
+      | 026210 | datumIngangFamilierechtelijkeBetrekking | ouder1 | ouder2       |
 
-Rule: in onderzoek wordt niet opgenomen wanneer het onderzoek beëindigd is
-  - Datum einde onderzoek (83.30) heeft een waarde
+    Abstract Scenario: persoon heeft <groep> <veld> van partner in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550002 |
+      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
+      | naam                          | waarde   |
+      | aanduidingGegevensInOnderzoek | <waarde> |
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | partners.<veld>                 |
+      Dan heeft de partner met burgerservicenummer '555550002' de volgende '<groep>.inOnderzoek' gegevens
+      | naam                 | waarde     |
+      | <in onderzoek>       | true       |
+      | datumIngangOnderzoek | 2022-03-07 |
+      En heeft de partner met burgerservicenummer '555550002' GEEN 'inOnderzoek' gegevens
+
+      Voorbeelden:
+      | waarde | veld                        | groep    |
+      | 050240 | geslachtsnaam               | naam     |
+      | 050310 | plaats                      | geboorte |
+      | 050610 | aangaanHuwelijkPartnerschap | datum    |
+
+  @proxy
+  Rule: wanneer een groep in de bron in onderzoek is, wordt elk gegeven in het antwoord dat gevuld wordt uit een van de elementen in die groep ook in inOnderzoek opgenomen met de waarde true
```

Comment:
'aanduidingGegevensInOnderzoek' heeft hier de waarde '010200' wat volgens mij betekent dat 

* naam.voornamen,
* naam.adellijkeTitelPredicaat,
* naam.voorvoegsel,
* naam.geslachtsnaam,
* naam.voorletters, 
* naam.volledigeNaam, 
* naam.aanhef, 
* naam.aanschrijfwijze,
* en naam.gebruikInLopendeTekst

inOnderzoek zijn. Moeten n.a.v. regel 12 niet ook de laatste 5 velden opgenomen worden in naam.inOnderzoek?
Of moeten alleen velden in naam.inOnderzoek worden opgenomen die met 'fields' zijn bevraagd?

Als dat laatste het geval is dan is mijn [eerste opmerking in deze PR](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/957/files#r827113104) waarschijnlijk niet van toepassing. Ik zou dan bovenin dit bestand nog wel het volgende toevoegen:

`Alleen een gegeven dat middels 'fields' expliciet wordt bevraagd kan in inOnderzoek worden opgenomen.`

Reply:
inderdaad, komt door de gebruikte fields